CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 4.16.0 (Unreleased)
+BREAKING CHANGES:
+* composer: made the `google_composer_environment.config.software_config.image_version` field immutable as this functionality is only available in beta. ([#11309](https://github.com/hashicorp/terraform-provider-google/pull/11309))
```

Comment:
@megan07 @ScottSuarez: The code change made suggests that _updating_ is only available in beta, is that right? The changelog here suggests the feature as a whole is beta-only.

Reply:
its available in the GA api. updated the comment correspondingly 

https://cloud.google.com/composer/docs/reference/rest/v1/projects.locations.environments